ddtrace/contrib/pylons/middleware.py

```diff
@@ -14,6 +19,12 @@ def __init__(self, app, tracer, service="pylons"):
         self._service = service
         self._tracer = tracer
 
+        # register middleware reference
+        config[CONFIG_MIDDLEWARE] = self
```

Comment:
we don't have circular references because the `PylonsTraceMiddleware` is never garbage collected.